ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml

```diff
@@ -45,33 +42,47 @@ tests:
     cluster_profile: medik8s-aws
     env:
       BASE_DOMAIN: medik8s-ci.projects.ecosystem.sysdeseng.com
-      INSTALL_NAMESPACE: self-node-remediation
+      OPERATOR_NAMESPACE: self-node-remediation
+      OPERATOR_RELEASED_VERSION: 0.8.0
```

Comment:
Just an idea, but we might upgrade the new-release.sh script (or to add a brand new one) to update this field in all files

Reply:
I tried it and running `make update` has removed it [`94a8675` (#52592)](https://github.com/openshift/release/pull/52592/commits/94a8675bea4f07f02358bf4c73dd16d4078c2018).

Are you suggesting that [new-release.sh](https://github.com/openshift/release/blob/master/ci-operator/config/medik8s/common/new-release.sh) will additionally **Y** bump this version (e.g., 0.8.0-> 0.9.0) ?

Reply:
Automating this would be awesome indeed. 
Printing a warning to not forget a manual update would be a first good step at least :)